functionaltest/src/test/resources/features/loginTest.feature

```diff
@@ -0,0 +1,59 @@
+@loginTest
+Feature: Login account
+  As a user
+  I want to be able to log into my account
+  In order to see my personal information
+
+  Background: Navigate to login page
+    Given I go to the page with URL "http://localhost:5000"
+
+  Scenario Outline: Successful Login
+    When I fill in "username" with "<username>"
+    And I fill in "password" with "<password>"
+    And I click on the "login" button
+    Then I should be on the page with URL "http://localhost:5000/account/view"
+    And I see in the field "username" there is "<usrname>"
```

Comment:
Could we use something like `"Username: <username>"`, instead of `"<usrname>"`?

Reply:
there is already <username> in a previous step, <usrname> is to avoid naming conflict. They takes different inputs so cannot be named as one.